torchvision/transforms/transforms.py

```diff
@@ -754,20 +758,53 @@ def get_params(brightness, contrast, saturation, hue):
             saturation in a random order.
         """
         transforms = []
-        if brightness > 0:
+        brightness_factor = None
+        if isinstance(brightness, float) and brightness >= 0:
```

Comment:
can you make this conditional into a helper method in this class? It has a lot of redundancy and if we ever want to update something, it gets more complicated.
Something in the lines of

```python
def _sample_from(self, value, name):
    factor = None
    if isinstance(value, numbers.Number) and value >= 0:
        ...
    elif isinstance(value, (tuple, list) len(value) == 2:
        ..
    else:
        raise TypeError('{} should be ...').format(name)
```